static/src/javascripts/projects/common/modules/commercial/donot-use-adblock.js

```diff
@@ -110,21 +133,31 @@ define([
                     linkText: 'Become a supporter from just £5 per month',
                     messageText: 'For just 16p a day you can support the Guardian\'s independence and our award-winning journalism',
                     template: 'coin'
+                },
+                {
+                    edition: 'INT',
+                    supporterLink: 'https://membership.theguardian.com/supporter?INTCMP=ADBLOCK_BANNER_COIN_INT',
+                    quoteText: 'Quality journalism',
+                    linkText: 'Become a supporter from just $4.99 per month',
+                    messageText: 'For just 16¢ a day you can support the Guardian\'s independence and our award-winning journalism',
+                    template: 'coin'
                 }
             ],
 
-            coinBannersRelevantToEdition = filter(coinBanners, function (message) {
-                return !message.edition || message.edition == config.page.edition;
-            }),
+            coinBannersRelevantToEdition = filter(coinBanners, relevantToEdition),
+            contributorBannersRelevantToEdition = filter(contributorBannersUK.concat(
+                map(contributorBannersUK, internationaliseBanner('INT')),
+                map(contributorBannersUK, internationaliseBanner('US'))
+            ), relevantToEdition),
```

Comment:
It seems that what you're really trying to do is create a set of contributorBanners that have their edition set to INT, UK or US if that's the edition we're currently reading - is that correct? It seems there might be a simpler way, e.g.

    function getContributorBanners() {
        var haveBanners = edition.match(/^(UK|US|INT)$/);
        return haveBanners ? map(banners, internationalise(edition)) : [];
    }